docker-entrypoint.sh

```diff
@@ -131,6 +131,15 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		export RABBITMQ_CTL_ERL_ARGS="$RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS"
 	fi
 
+	echo
+	for f in /docker-entrypoint-initdb.d/*; do
+		case "$f" in
```

Comment:
@md5 Is the sourcing a problem? I guess that the script could affect the environment is a good thing? More power to extend the base docker?